apps/studio/components/interfaces/App/FeaturePreview/FeaturePreview.constants.tsx

```diff
@@ -10,7 +10,7 @@ export const FEATURE_PREVIEWS = [
   },
   {
     key: LOCAL_STORAGE_KEYS.UI_PREVIEW_BRANCHING_2_0,
-    name: 'Branching 2.0',
+    name: 'Create and merge branches via dashboard',
```

Comment:
just a nit here - if we might be able to make the title shorter

<img width="600" height="213" alt="image" src="https://github.com/user-attachments/assets/fa7f970e-1e0f-4407-822e-b380f51dbaad" />